docs/validators/sigstore_cosign.md

```diff
@@ -97,6 +97,7 @@ kubectl run altsigned --image=docker.io/securesystemsengineering/testimage:co-si
 | `auth.` | | | Authentication credentials for private registries. |
 | `auth.secret_name` | | | Name of a Kubernetes secret in Connaisseur namespace that contains [dockerconfigjson](https://kubernetes.io/docs/concepts/configuration/secret/#docker-config-secrets) for registry authentication. See additional notes [below](#authentication). |
 | `cert` | | | A certificate in PEM format for private registries. |
+| `k8s_keychain` | false | | When true, pass `--k8s-keychain` argument to `cosign verify`. |
```

Comment:
key value might need to be adjusted based on the discussion above to `auth.k8s_keychain`. Furthermore, would like to be a bit more explicit in what it does:
```suggestion
| `auth.` | | | Authentication credentials for private registries. See additional notes [below](#authentication). |
| `auth.secret_name` | | | Name of a Kubernetes secret in Connaisseur namespace that contains [dockerconfigjson](https://kubernetes.io/docs/concepts/configuration/secret/#docker-config-secrets) for registry authentication. See additional notes [below](#dockerconfigjson). |
| `auth.k8s_keychain` | false | | When true, pass `--k8s-keychain` argument to `cosign verify` in order to use workload identities for authentication. See additional notes [below](#k8s_keychain). |
| `cert` | | | A certificate in PEM format for private registries. |
```

Reply:
I used your suggested changes. Thanks!